packages/odyssey-react-mui/src/labs/SideNav/SideNav.tsx

```diff
@@ -212,34 +310,41 @@ const SideNav = ({
   const sideNavStyles = useMemo(
     () => ({
       display: "flex",
-      height: "100vh",
+      height: "100%",
+      maxWidth: expandedWidth,
+      overflow: "hidden",
     }),
-    [],
+    [expandedWidth],
   );
```

Comment:
We have a mix of styled components and `Box` styles. Is it possible we could use only styled components the whole way? Those components can take props, so you'd pass these in as props rather than as a `style` object. Should make it so we don't have two ways of styling components in here.

Reply:
I didn't like that there was a mix and was converting to styled when it seemed to make sense. If you're open to this ballooning, I can convert all of them